### ISSUE TITLE
Styles used in properties that are not named style are incorrectly marked as unused

### ISSUE THREAD

**Issue by Intellicode** [Intellicode/eslint-plugin-react-native#2]
When style properties are used in JSX in non-style tags, they are marked as unused:

```
var styles = StyleSheet.create({ 
   button: { color: 'red' }
});

var Hello = React.createClass({
  render() { return <SomeComponent buttonStyle={styles.button />; }
});
```
